notebooks/scripts/load_pretrained_weights.py

```diff
@@ -0,0 +1,26 @@
+#Loads the downloaded pretrained weights into the specified model.
+import torch
+
+def load_weights(Model, Mode, Setting, model_object):
+    if Mode=='3D':
+        if Setting=='4class':
+            checkpoint = torch.load(str(Model)+'_'+str(Mode)+'_noZI.pt', map_location='cuda:0')
```

Comment:
do these lines with cuda:0 work on cpu? The notebook suggests this will, but I want to confirm- seems to assume a cuda device.